.travis.yml

```diff
@@ -1,17 +1,68 @@
+sudo: false
 language: cpp
 
 compiler:
   - gcc
-  - clang
 
 env:
+  matrix:
+    - FUZZ=1
+    - NO_INLINE=1
+    - NO_BIT=1
+    - STATIC=1 NO_DEBUG=1
+    - SAFETY_PROFILE=1
+    - MOD_DEFAULT_FEATURES=1 USES_WCHAR=1
```

Comment:
Was using this flag to simplify the construction of all the ADDL_DEFAULT_FEATURES additions that would need to be included after the --features="..." config param.  If we eventually want to do more than a wchar build here this would be beneficial.